similar/main/multi.cpp

```diff
@@ -1041,12 +1041,7 @@ void multi_do_frame(void)
 
 	multi_do_protocol_frame(0, 1);
 
-	if (multi_quit_game)
-	{
-		multi_quit_game = 0;
-		if (Game_wind)
-			window_close(Game_wind);
-	}
+	return multi_quit_game ? window_event_result::close : window_event_result::handled;
```

Comment:
Yes, I intended this change. This is in case `multi_quit_game` is first caught other than in the call to `multi_do_frame()` in `GameProcessFrame()`.